src/pages/Login/Login.jsx

```diff
@@ -1,13 +1,106 @@
+import "./Login.css"
 import { Navbar } from "../../components/Navbar/Navbar";
+import { useState } from "react";
+import { useAuth } from "../../context/authContext";
+import { loginService } from "../../services/loginService";
+import { Link } from "react-router-dom";
 
 const Login = () => {
+
+  const [loginInput, setLoginInput] = useState({});
+  const [hide, setHide] = useState({ pswd: true });
+
+  const { setIsAuth, setToken, navigate } = useAuth();
+
+  const loginInputHandler = (e) => {
+    const { name, value } = e.target;
+    setLoginInput({ ...loginInput, [name]: value });
+  };
+
+  const loginHandler = async (e) => {
+    e.preventDefault();
+
+    try {
+      const { data } = await loginService(loginInput);
+
+      localStorage.setItem("token", data.encodedToken);
+      setToken(data.encodedToken);
+      setLoginInput({ email: "", password: "" });
+      setIsAuth(true);
+      navigate("/");
+    } catch (err) {
+      console.log(err);
+    }
+  };
+
   return (
     <div className="page-wrapper">
       <Navbar />
 
-      <section>
-        <h1>Login Page</h1>
-      </section>
+      <main>
+      <div className="login-section">
+
+          <div className="login-heading">
+              <h2>Login</h2>
+          </div>
+
+          <div className="login-input">
+          <form onSubmit={loginHandler}>
+
+              <div className="username-box">
+                  <label for="email">Email:</label>
```

Comment:
You can replace it with react `htmlFor` attribute

Reply:
I will in the end when i refactor the code, thankyou for reviewing it